regression.sh

```diff
@@ -28,10 +28,19 @@ rm -f "$RESULT_DIR"/*.md
 
 echo "## \`fd\` regression benchmark" >> "$REPORT"
 
-heading "Interactive output"
+heading "Interactive output (colored)"
 hyperfine --warmup "$WARMUP_COUNT" \
-    "$FD1 '' '$SEARCH_ROOT'" \
-    "$FD2 '' '$SEARCH_ROOT'" \
+    "$FD1 '' '$SEARCH_ROOT' --color=always" \
+    "$FD2 '' '$SEARCH_ROOT' --color=always" \
+    --show-output \
+    --export-markdown "$RESULT_DIR/00-interactive-output.md" \
+    --export-json "$RESULT_DIR/00-interactive-output.json"
+cat "$RESULT_DIR/00-interactive-output.md" >> "$REPORT"
```

Comment:
these write to the same files as below. Any chance we can follow the existing pattern (with increasing numbers for each benchmark) here and also keep all intermediate results (especially the JSON files)?